lib/dradis-csv.rb

```diff
@@ -1,3 +1,5 @@
+require 'csv'
```

Comment:
Is this needed? I thought rails has csv loaded by default.

Reply:
I'd say yes since this errors-out without this line. Not sure why exactly